pages/board_page_server.R

```diff
@@ -0,0 +1,43 @@
+
+####################### Boards Page - SERVER #######################
+
+
+
+filtered_data <- reactive({
+  Join_all_long2 %>%
+    filter(`NHS Board` == input$board) %>%
+    filter(Specialty == input$specialty)
+})
+
+
+
+#waiting-list_chart
+
+
+output$waiting_list_chart <- renderPlot({
+    ggplot(data = filtered_data(),
+         aes(x = `Month/Year`, y = Value, group = Indicator)) +
+    geom_line(aes(colour= Indicator), size=1.2) +
+    theme_bw() +
+    theme(legend.text = element_text(size = 12)) +
+    theme(legend.position="bottom") +
+    theme(legend.title=element_blank()) +
+    theme(axis.text=element_text(size=12),axis.title=element_text(size=12)) +
+    # scale_x_date(date_labels = "%b %y", breaks = as.Date(c("2019-01-01", "2019-04-01",
+    #                                                        "2019-07-01", "2019-10-01",
```

Comment:
Could you get rid of these lines now? It'll help keep your code tidy if you remove redundant commented-out sections as you go.